Jenkinsfile

```diff
@@ -73,9 +73,14 @@ pipeline {
           withBeatsEnv(archive: false, id: "lint") {
             dumpVariables()
             setEnvVar('VERSION', sh(label: 'Get beat version', script: 'make get-version', returnStdout: true)?.trim())
-            cmd(label: "make check-python", script: "make check-python")
-            cmd(label: "make check-go", script: "make check-go")
-            cmd(label: "Check for changes", script: "make check-no-changes")
+            whenTrue(env.ONLY_DOCS == 'true') {
+              cmd(label: "make check", script: "make check")
```

Comment:
Is possible to run the lint stage(Jenkinsfile.yml) of each beat? it is the same but in parallel so faster

Reply:
That's another approach to solve this issue:

* Either to exclude the ONLY_DOCS when in the Builds&Tests dynamic stages
* Or this approach

That's the current implementation, but my aim with this PR was to avoid changing the current implementation but enabling the `make check` only for the DOCS changes. 

In both cases, the output is the same, I guess

Reply:
let's live with it, but takes 15 min :(

Reply:
It's only for `docs` changes, if any kind of code changes, then it will work as used to be.